aiida/cmdline/commands/cmd_restapi.py

```diff
@@ -26,34 +26,44 @@
 
 
 @verdi.command('restapi')
-@click.option('-H', '--host', type=click.STRING, default='127.0.0.1', help='the hostname to use')
-@click.option('-p', '--port', type=click.INT, default=5000, help='the port to use')
+@click.option('-H', '--hostname', type=click.STRING, default='127.0.0.1', help='Hostname')
```

Comment:
please use the predefined `aiida.cmdline.params.options.HOSTNAME` and `aiida.cmdline.params.options.PORT` for these. Note you can override the default when you call it. See other verdi commands for examples

Reply:
I have made changes for this but I am not sure if it is correct. Please let me know if I am missing anything again.

Reply:
nope, all good :+1: